src/main/java/org/ibp/api/brapi/v2/germplasm/BreedingMethodResourceBrapi.java

```diff
@@ -0,0 +1,98 @@
+
+	@Autowired
+	private BreedingMethodService breedingMethodService;
+
+	@ApiOperation(value = "Get the Breeding Methods", notes = "Get the list of germplasm breeding methods available in a system.")
+	@RequestMapping(value = "/{crop}/brapi/v2/breedingmethods", method = RequestMethod.GET)
+	@ResponseBody
+	public ResponseEntity<EntityListResponse<BreedingMethod>> getAllBreedingMethods(@PathVariable final String crop, @ApiParam(value = BrapiPagedResult.CURRENT_PAGE_DESCRIPTION)
+	@RequestParam(value = "page", required = false) final Integer currentPage,
+		@ApiParam(value = BrapiPagedResult.PAGE_SIZE_DESCRIPTION)
+		@RequestParam(value = "pageSize", required = false) final Integer pageSize) {
+		final PagedResult<BreedingMethod> resultPage = new PaginatedSearch().executeBrapiSearch(currentPage, pageSize,
+			new SearchSpec<BreedingMethod>() {
+
+				@Override
+				public long getCount() {
+					return BreedingMethodResourceBrapi.this.breedingMethodService.getAllBreedingMethods().size();
+				}
+
+				@Override
+				public List<BreedingMethod> getResults(final PagedResult<BreedingMethod> pagedResult) {
+					final List<BreedingMethod> results = new ArrayList<>();
+					final List<BreedingMethodDTO> breedingMethodDTOS = BreedingMethodResourceBrapi.this.breedingMethodService.getAllBreedingMethods();
+					for (final BreedingMethodDTO dto : breedingMethodDTOS) {
+						final BreedingMethod method = new BreedingMethod();
+						method.setAbbreviation(dto.getCode());
+						method.setBreedingMethodDbId(String.valueOf(dto.getMid()));
+						method.setBreedingMethodName(dto.getName());
+						method.setDescription(dto.getDescription());
+						results.add(method);
+					}
+					return results;
+				}
+			});
+		final Result<BreedingMethod> result = new Result<BreedingMethod>().withData(resultPage.getPageResults());
+		final Pagination pagination = new Pagination().withPageNumber(resultPage.getPageNumber()).withPageSize(resultPage.getPageSize())
+			.withTotalCount(resultPage.getTotalResults()).withTotalPages(resultPage.getTotalPages());
+		final Metadata metadata = new Metadata().withPagination(pagination);
+		final EntityListResponse<BreedingMethod> entityListResponse = new EntityListResponse<BreedingMethod>(metadata, result);
+		return new ResponseEntity<>(entityListResponse, HttpStatus.OK);
+	}
+
+	@ApiOperation(value = "Get the details of a specific Breeding Method used to produce Germplasm", notes = "Get the details of a specific Breeding Method")
+	@RequestMapping(value = "/{crop}/brapi/v2/breedingmethod/{breedingMethodDbId}", method = RequestMethod.GET)
+	@ResponseBody
+	public ResponseEntity<SingleEntityResponse<BreedingMethod>> getBreedingMethodById(
+		@PathVariable final String crop,
+		@PathVariable final Integer breedingMethodDbId) {
+
+
+		final BreedingMethodDTO breedingMethodDTOS = this.breedingMethodService.getBreedingMethod(breedingMethodDbId);
+		final BreedingMethod method = new BreedingMethod();
```

Comment:
Same here, use use `ModelMapper `to convert from BreedingMethodDTO to BreedingMethod